ppr-ui/src/components/mhrRegistration/HomeOwners/FractionalOwnership.vue

```diff
@@ -122,11 +131,15 @@ export default defineComponent({
 </script>
 
 <style lang="scss" scoped>
+.backgroundWhite {
```

Comment:
`.background-white` 
Kebab case 👍

Reply:
fixed.  old habits die hard...

Reply:
this is a weird one... I changed to background-white and class and style bindings stopped working. ie would not compile..  I changed to backgroundwhite and it works...